waku/v2/protocol/waku_swap/waku_swap.nim

```diff
@@ -103,6 +145,15 @@ proc init*(wakuSwap: WakuSwap) =
       wakuSwap.accounting[peerId] = n
     info "Accounting state", accounting = wakuSwap.accounting[peerId]
 
+    # TODO Isolate to policy function
+    # TODO Tunable payment threshhold, hard code for PoC
+    let paymentThreshhold = 1
+    if wakuSwap.accounting[peerId] >= paymentThreshhold:
```

Comment:
Ok, so for now the peer takes no action to "remind" the indebted node to send a cheque or agree on the outstanding balance. Do we foresee persistence issue in future where peers have differing views on outstanding balances after a restart, or will this be a non-issue when contracts are specified?

Reply:
TBD, some persistence etc might be needed indeed. In general, it is useful if both peers agree but if one peer is misbehaving/dishonest that shouldn't break things, conceptually.